supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionCommands.scala

```diff
@@ -15,15 +17,26 @@ object ActionCommands {
   implicit val nbRetries: Int = 2
   implicit val waitBetweenRetries: FiniteDuration = 5.seconds
 
+  implicit class EnhancedCromwellPath(val path: Path) extends AnyVal {
+    def projectId: String = path match {
+      case gcs: GcsPath => gcs.projectId
+      case _ => ""
+    }
+  }
+  
   implicit class ShellPath(val path: Path) extends AnyVal {
     // The command String runs in Bourne shell so shell metacharacters in filenames must be escaped
     def escape = ESCAPE_XSI.translate(path.pathAsString)
   }
+  
+  private def makeContentTypeFlag(contentType: Option[ContentType]) = contentType.map(ct => s"""-h "Content-Type: $ct"""").getOrElse("")
 
   def makeContainerDirectory(containerPath: Path) = s"mkdir -p ${containerPath.escape}"
 
-  def delocalizeDirectory(containerPath: Path, cloudPath: Path) = retry {
-    s"gsutil -m rsync -r ${containerPath.escape} ${cloudPath.escape}"
+  def delocalizeDirectory(containerPath: Path, cloudPath: Path, contentType: Option[ContentType]) = retry {
+    recoverRequesterPaysError(cloudPath) { flag =>
+      s"gsutil $flag ${contentType |> makeContentTypeFlag} -m rsync -r ${containerPath.escape} ${cloudPath.escape}"
```

Comment:
I kinda like it in string interpolations because it removes a pair of parentheses where it's already wrapped in `${}`

```scala
s"${makeContentTypeFlag(contentType)}"

s"${contentType |> makeContentTypeFlag}"
```
I could be convinced otherwise though